client/src/app/components/FilterToolbar/MultiselectFilterControl.tsx

```diff
@@ -357,6 +357,7 @@ export const MultiselectFilterControl = <TItem,>({
       id={`filter-control-${category.key}`}
       chips={chips}
       deleteChip={(_, chip) => onFilterClear(chip)}
+      deleteChipGroup={() => setFilterValue([])}
```

Comment:
Passing thought -- to be more consistent with the deleteChip prop, add a `onFilterClearAll()` function and put it right next to `onFilterClear` so the actual work of clearing the filter values are in the same place.